### ISSUE TITLE
Commit window not open

### ISSUE THREAD

**Issue by gollum23** [syuchan1005/GitPrefix#9]
Hi, i just now update git prefix plugin and the commit window not open, here is the stack trace error
```
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - Argument for @NotNull parameter 'dimension' of com/intellij/util/ui/JBInsets.removeFrom must not be null 
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'dimension' of com/intellij/util/ui/JBInsets.removeFrom must not be null
	at com.intellij.util.ui.JBInsets.$$$reportNull$$$0(JBInsets.java)
	at com.intellij.util.ui.JBInsets.removeFrom(JBInsets.java)
	at com.intellij.ide.ui.laf.darcula.ui.DarculaRadioButtonUI.updatePreferredSize(DarculaRadioButtonUI.java:104)
	at com.intellij.ide.ui.laf.darcula.ui.DarculaRadioButtonUI.getPreferredSize(DarculaRadioButtonUI.java:99)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1663)
	at com.intellij.openapi.ui.VerticalFlowLayout.preferredLayoutSize(VerticalFlowLayout.java:165)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at com.intellij.ui.components.JBViewport.getPreferredSizeWithoutScrollBars(JBViewport.java:531)
	at com.intellij.ui.components.JBViewport.getPreferredScrollableViewportSize(JBViewport.java:526)
	at com.intellij.ui.components.JBViewport.access$100(JBViewport.java:30)
	at com.intellij.ui.components.JBViewport$1.preferredLayoutSize(JBViewport.java:63)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at com.intellij.ui.components.JBScrollPane$Layout.preferredLayoutSize(JBScrollPane.java:689)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at com.intellij.openapi.ui.Splitter.getPreferredSize(Splitter.java:216)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at com.intellij.openapi.ui.Splitter.getPreferredSize(Splitter.java:217)
	at com.intellij.openapi.vcs.changes.ui.CommitChangeListDialog$MyOptionsLayout.preferredLayoutSize(CommitChangeListDialog.java:1234)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at com.intellij.ui.components.JBPanel.getPreferredSize(JBPanel.java:122)
	at com.intellij.openapi.ui.Splitter.getPreferredSize(Splitter.java:216)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at javax.swing.JRootPane$RootLayout.preferredLayoutSize(JRootPane.java:920)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at java.awt.BorderLayout.preferredLayoutSize(BorderLayout.java:719)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at java.awt.Window.pack(Window.java:809)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:632)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:430)
	at com.intellij.openapi.ui.DialogWrapper.invokeShow(DialogWrapper.java:1690)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1639)
	at com.intellij.openapi.vcs.changes.ui.CommitChangeListDialog.commitChanges(CommitChangeListDialog.java:233)
	at com.intellij.openapi.vcs.changes.ui.CommitChangeListDialog.commitChanges(CommitChangeListDialog.java:154)
	at com.intellij.openapi.vcs.actions.AbstractCommonCheckinAction.performCheckIn(AbstractCommonCheckinAction.java:87)
	at com.intellij.openapi.vcs.actions.AbstractCommonCheckinAction.a(AbstractCommonCheckinAction.java:65)
	at com.intellij.openapi.vcs.changes.Waiter.onSuccess(Waiter.java:65)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:494)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressSynchronously(CoreProgressManager.java:449)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressSynchronously(ProgressManagerImpl.java:109)
	at com.intellij.openapi.progress.impl.CoreProgressManager.b(CoreProgressManager.java:318)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:303)
	at com.intellij.openapi.vcs.changes.CallbackData.b(CallbackData.java:85)
	at com.intellij.openapi.vcs.changes.UpdateRequestsQueue.invokeAfterUpdate(UpdateRequestsQueue.java:196)
	at com.intellij.openapi.vcs.changes.ChangeListManagerImpl.invokeAfterUpdate(ChangeListManagerImpl.java:410)
	at com.intellij.openapi.vcs.changes.ChangeListManagerImpl.invokeAfterUpdate(ChangeListManagerImpl.java:398)
	at com.intellij.openapi.vcs.actions.AbstractCommonCheckinAction.actionPerformed(AbstractCommonCheckinAction.java:65)
	at com.intellij.openapi.vcs.actions.AbstractVcsAction.actionPerformed(AbstractVcsAction.java:48)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.actionSystem.impl.ActionButton.a(ActionButton.java:203)
	at com.intellij.openapi.actionSystem.impl.ActionButton.a(ActionButton.java:145)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:386)
	at java.awt.Component.processEvent(Component.java:6313)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4903)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4526)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.a(IdeEventQueue.java:781)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:718)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - PyCharm 2018.2  Build #PY-182.3684.100 
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - JDK: 1.8.0_152-release 
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - VM: OpenJDK 64-Bit Server VM 
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - Vendor: JetBrains s.r.o 
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - OS: Linux 
2018-07-29 15:49:36,510 [  96542]  ERROR - llij.ide.plugins.PluginManager - Last Action: CheckinProject 
2018-07-29 15:49:36,524 [  96556]  ERROR - llij.ide.plugins.PluginManager - Argument for @NotNull parameter 'dimension' of com/intellij/util/ui/JBInsets.removeFrom must not be null 
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'dimension' of com/intellij/util/ui/JBInsets.removeFrom must not be null
	at com.intellij.util.ui.JBInsets.$$$reportNull$$$0(JBInsets.java)
	at com.intellij.util.ui.JBInsets.removeFrom(JBInsets.java)
	at com.intellij.ide.ui.laf.darcula.ui.DarculaRadioButtonUI.updatePreferredSize(DarculaRadioButtonUI.java:104)
	at com.intellij.ide.ui.laf.darcula.ui.DarculaRadioButtonUI.getPreferredSize(DarculaRadioButtonUI.java:99)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1663)
	at com.intellij.openapi.ui.VerticalFlowLayout.preferredLayoutSize(VerticalFlowLayout.java:165)
	at java.awt.Container.preferredSize(Container.java:1797)
	at java.awt.Container.getPreferredSize(Container.java:1781)
	at javax.swing.JComponent.getPreferredSize(JComponent.java:1665)
	at com.intellij.ui.components.JBScrollPane$Layout.layoutContainer(JBScrollPane.java:426)
	at java.awt.Container.layout(Container.java:1511)
	at java.awt.Container.doLayout(Container.java:1500)
	at java.awt.Container.validateTree(Container.java:1696)
	at java.awt.Container.validateTree(Container.java:1705)
	at java.awt.Container.validateTree(Container.java:1705)
	at java.awt.Container.validateTree(Container.java:1705)
	at java.awt.Container.validateTree(Container.java:1705)
	at java.awt.Container.validateTree(Container.java:1705)
	at java.awt.Container.validateTree(Container.java:1705)
	at java.awt.Container.validate(Container.java:1631)
	at com.intellij.ui.SplitterWithSecondHideable$MyTitledSeparator.a(SplitterWithSecondHideable.java:112)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.a(IdeEventQueue.java:781)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:722)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2018-07-29 15:49:36,524 [  96556]  ERROR - llij.ide.plugins.PluginManager - PyCharm 2018.2  Build #PY-182.3684.100 
2018-07-29 15:49:36,525 [  96557]  ERROR - llij.ide.plugins.PluginManager - JDK: 1.8.0_152-release 
2018-07-29 15:49:36,525 [  96557]  ERROR - llij.ide.plugins.PluginManager - VM: OpenJDK 64-Bit Server VM 
2018-07-29 15:49:36,525 [  96557]  ERROR - llij.ide.plugins.PluginManager - Vendor: JetBrains s.r.o 
2018-07-29 15:49:36,525 [  96557]  ERROR - llij.ide.plugins.PluginManager - OS: Linux 
2018-07-29 15:49:36,525 [  96557]  ERROR - llij.ide.plugins.PluginManager - Last Action: CheckinProject 
```

**Comment by syuchan1005** [comment]
It worked on my Windows, but it did not work on Mac.
I will fix it but it will take time.

**Comment by syuchan1005** [comment]
Sorry, I lied to you.
It seems to happen with the Darcula theme. 
Please temporarily use IntelliJ's theme.

**Comment by u0xaa55** [comment]
I'm on a mac but I'm not using Darcula theme. If it's also worth to notice: I installed the plugin on WebStorm

**Comment by syuchan1005** [comment]
@astrinxit66 
Same exception?  
Are you using the IntelliJ theme?

**Comment by syuchan1005** [comment]
Fixed the problem with version 1.5.3.1.
If it occurs again, reopen it.
